### PR TITLE
KEYCLOAK-4101: jboss-cli script to do migration of configuration

### PR DESCRIPTION
**Pull request by ssilvert**
This version moves the migration tests out of testsuite and under distribution.  This is probably where it belongs anyway since the migration scripts are part of the distribution and not part of Keycloak proper.

The test relies on server-dist being built already, so there was no way it was going to work under testsuite anyway.

### CODE REVIEW

**Comment by ssilvert** [comment]
Stian, the tests will work if you run each job individually.  

We may need to discuss this anyway.  The tests are in a new module off of /distribution.  They can't go in /testsuite because it relies on server-dist, which is not built yet.  And, the thing being tested is CLI scripts which are part of the feature-packs.  So I do think /distribution is the right place for the tests.

One thing I don't understand is how the build is broken into different jobs.  The new tests I've added only need to be run once, but they appear to be running with each job, which causes a conflict.

**Comment by stianst** [comment]
I assume https://github.com/keycloak/keycloak/pull/3660 is not relevant anymore and should be closed?

The tests belong in testsuite and there's nothing wrong on having that depend on server-dist. You can simply make the tests only run when -Pauth-server-wildfly is included.

**Comment by stianst** [comment]
@ssilvert  I moved our biweekly to Monday and we can discuss this then

**Comment by ssilvert** [comment]
That should work as long as it runs by default.  The whole point of the test is really to alert developers when they change standalone.xml but forget to update the migration scripts.

**Comment by stianst** [comment]
Let's discuss on Monday if it should run by default or as part of CI. Not everything can run by default and on Travis as we need to consider time it takes to run everything. Some things will be caught by ~nightly builds which we will put a process in place to review and have the person doing the reviewer identify who broke it.

**Comment by ssilvert** [comment]
Actually, #3660 is the one we will probably end up merging.  That one runs from testsuite.  I just need to make a few updates.

**Comment by ssilvert** [comment]
We'll merge this one after all.

File: testsuite/integration-arquillian/tests/other/pom.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
     <modules>
         <module>adapters</module>
-        <module>sssd</module>
+        <!--<module>sssd</module>-->
```
Comment (stianst) [inline testsuite/integration-arquillian/tests/other/pom.xml]: I assume commenting out this module is a mistake?

**Comment by ssilvert** [comment]
Oops!  Yes, that's a mistake.  I'll fix it ASAP.

**Pull request merged**